src/components/header/header.module.css

```
@@ -1,4 +1,128 @@
-.header {
-  font-size: 34px;
-  text-align: center;
+.headerContainer {
+  display: flex;
+  align-items: center;
+  min-height: 80px;
+}
+
+.logo {
+  flex-grow: 2;
+  max-height: 46px;
+}
+
+.appLayout {
+  flex-grow: 20;
+}
+
+.navBar {
+  flex-grow: 2;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.activeNavBarItem {
+  margin-right: 40px;
+  display: flex;
+  padding: 0;
+  border: none;
+  background: transparent;
+  color: #FFF;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 20px;
+}
+
+.navBarItem {
+  margin-right: 30px;
+  display: flex;
+  padding: 0;
+  border: none;
+  background: transparent;
+  color: #FFF;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 20px;
+}
+
+.actionItems {
+  flex-grow: 4;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.sun {
+  background: transparent;
+  padding: 10px 0 10px 5px;
+}
+
+.moon {
+  background: transparent;
+  padding: 10px 5px;
+}
+
+button.sun::after {
```

```suggestion
.sun::after {
```
Try to avoid unnecessary specifics in CSS. In the future it can be harder to override

---

Fixed